docs/section5/Rule5-43.md

```diff
@@ -0,0 +1,43 @@
+
+# Envelope - Rule 5-43  
+
+**Rule ID:** 5-43  
+**Rule Description:** The  proposed roof surfaces shall be modeled using the same solar reflectance as in the user model.  
+**Rule Assertion:** P-RMR SurfaceOpticalProperties:absorptance_solar_exterior = U-RMR SurfaceOpticalProperties:absorptance_solar_exterior  
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-1(a) Building Envelope Modeling Requirements for the Proposed design  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. get_opaque_surface_type()
+  2. match_data_element()
+
+## Rule Logic:  
+
+- For each building segment in the Proposed model: `for building_segment_p in P_RMR.building.building_segments:`
+
+  - For each thermal block in building segment: `for thermal_block_p in building_segment_p.thermal_blocks:`
+
+    - For each zone in thermal block: `for zone_p in thermal_block_p.zones:`
+
+      - For each surface in zone: `for surface_p in zone_p.surfaces;`
+
+        - Check if surface is roof, get matching surface in U_RMR: `if get_opaque_surface_type(surface_p.id) == "ROOF": surface_u = match_data_element(U_RMR, Surfaces, surface_p.id)`
+
+          **Rule Assertion:**  
+
+          - Case 1: If roof surface solar reflectance in P_RMR matches U_RMR and is equal to 0.3: `if ( surface_p.surface_optical_properties.absorptance_solar_exterior == surface_u.surface_optical_properties.absorptance_solar_exterior ) AND ( surface_p.surface_optical_properties.absorptance_solar_exterior == 0.7 ): PASS`
+
+          - Case 2: Else if roof surface solar reflectance in P_RMR matches U_RMR and is not equal to 0.3: `else if ( surface_p.surface_optical_properties.absorptance_solar_exterior == surface_u.surface_optical_properties.absorptance_solar_exterior ) AND ( surface_p.surface_optical_properties.absorptance_solar_exterior != 0.7 ): PASS and raise_warning "ROOF SURFACE SOLAR REFLECTANCE IN P-RMR MATCHES THAT IN U-RMR BUT IS NOT EQUAL TO 0.3.`
+
```

Comment:
What does "raise_warning" mean? I think this should either be a message accompanying "PASS" or we should change outcome to "CAUTION". I don't have strong feelings one way or the other, perhaps with slight preference for CAUTION since the rule does not follow general logic of p-RMR = B-RMR. I would also change the message to say  "ROOF SURFACE SOLAR REFLECTANCE IN P-RMR MATCHES THAT IN U-RMR BUT IS NOT EQUAL TO 0.3. Verify that reflectance was established using aged test data as required in Section 5.5.3.1(a)`

Reply:
Updated Case 2 to Caution.

Reply:
@mkarpman why does an unregulated surface need to have the specified value? an unregulated surface could have any value in the U-RMR and be different from the P-RMR which should be 0.3. I don't think we need a CAUTION here, it could still be pass.

Reply:
Discussed, this needs to be updated to verify that it is the same between U and P for an unregulated surface

Reply:
Updated. Please confirm if the rule assertions are correct.